screens/ContactScreen.js

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import { View, Text, TextInput, StyleSheet, Button } from 'react-native';
+import styled from 'styled-components/native'
+
+
+const ContactContainer = styled.View`
+  display: flex;
+  align-items: center;
+  height: 100%;
+`;
+
+
+export const Contact = () => {
+  const [Name, onChangeName] = React.useState(null);
+  const [Email, onChangeEmail] = React.useState(null);
+  const [number, onChangeNumber] = React.useState(null);
+  const [Text, onChangeText] = React.useState(null);
+  
+  return (
+    <ContactContainer>
+      <h1>Contact Us</h1>
+      <TextInput
+        style={styles.input}
+        onChangeText={onChangeName}
+        placeholder="Name"
+        value={Name}
+        keyboardType="default"
```

Comment:
I never used keyboardType, I guess it is chosing default keyboard type in your phone - cool!!